src/systems/systems/src/air_conditioning/mod.rs

```diff
@@ -173,6 +175,11 @@ impl<const ZONES: usize> AirConditioningSystemOverhead<ZONES> {
         [self.pack_1_pb.is_on(), self.pack_2_pb.is_on()]
     }
 
+    fn set_pack_pushbutton_fault(&mut self, pb_has_fault: [bool; 2]) {
+        self.pack_1_pb.set_fault(pb_has_fault[0]);
+        self.pack_2_pb.set_fault(pb_has_fault[1]);
+    }
```

Comment:
**suggestion**: Do consider changing `pack_1_pb` and `pack_2_pb` to something like `pack_pbs: [OnOffFaultPushButton; 2]`. Might lead to better code.

Reply:
Done!